docs/forms.md

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_label: "Forms"
+sidebar_position: 4
+---
+
+# Forms
+
+After finishing setting up flutter and stacked (if you didn't setup stacked check out [get-started](get-started.md)), one of the features stacked provides is form handling by connecting form data in view and view model so that business logic and UI are separate and consistent with our goal. 
```

Comment:
, one of the features stacked provides is form handling. We automatically sync the values from the controller with the ViewModel reducing the amount of boiler plate required to manage forms.